stage1/usr_from_coreos/cache.sh

```diff
@@ -10,332 +10,172 @@ if [ ${V} -eq 3 ]; then
 	set -x
 fi
 
-# coreos gpg signing key
```

Comment:
This is a pretty big change, we should justify it a bit more in the commit message. I propose something like this:

```
As CoreOS Container Linux is in maintenance mode
(https://github.com/coreos/bugs/issues/2559#issuecomment-469250304) and won't
see many new features and we need some systemd bugfixes that rkt needs
(specifically https://github.com/systemd/systemd/pull/13173 and
https://github.com/systemd/systemd/pull/12860) let's switch to Flatcar Linux.

Flatcar Linux us a drop-in replacement of Container Linux and its Edge channel
contains the needed bugfixes.

To do that, we need to change the GPG key to verify the images through the script.
```

Also, we can keep all the Container Linux -> Flatcar changes in one commit.